README.md

```diff
@@ -52,6 +52,32 @@ any suggestions! All of the software here is written by students.
 10. Point your browser to [`http://localhost:3000`](http://localhost:3000)
     :heart_eyes:
 
+##### Windows Alternative w/ WSL Ubuntu
+
+1. Install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10). Please download Ubuntu.
+2. Open Ubuntu and ensure git is installed by running `git --version`.
+3. Navigate back to the Windows c drive with `cd /mnt/c/Users`.
+4. Now once you are here, cd into your own user folder. Then navigate to a directory that you want to work in.
+   (Do not just put it in your user folder--go to your desktop or documents folder)
+   Ex) `cd Bob/Desktop` and `mkdir radio_stuff`
+5. Run the command `git clone https://github.com/uclaradio/uclaradio.git`. While you wait for it download,
+   give us some love by starring our repo.
+6. Navigate to the set up script with `cd uclaradio/scripts`.
+   Run `sudo apt-get install dos2unix` and then `dos2unix setup_ubuntu.sh` (to fix WSL formatting [error](https://askubuntu.com/questions/966488/how-do-i-fix-r-command-not-found-errors-running-bash-scripts-in-wsl)).
```

Comment:
I didn't need to run this. Might be better to make this an aside, with something along the lines of "If you run into so-and-so error, run these commands".

Reply:
hmmm that's probs cause i pushed after formatting so github must've "preserved" it XD? i'll switch it